1.8/administration/categories-and-forums.md

```diff
@@ -0,0 +1,54 @@
+For example, we have **category** named Community and under it **forums** like General Discussion, Showcase etc.
+
+### What Category can contain?
+
+Well, when creating new category they need to contain:
+- **Forum Type** - Forum type should be **category**.
+- **Title** - name of your category. Multiple categories can have same name. It is not it's id.
+- **Description** - You can have Category description. That can be few words wich will be displayed right below Category name at forum's home page. This field is **not** necessary and is mostly blank.
+- **Parent Forum** - Mostly this is left **None** as categories are here to actually be parents. But if you wish, you can put it under another category. Then it will look like forum but you will be unaviable to write in it so there is not much point in doing that.
+- **Display order** - This is position of your category between other categories. So if you have more than one category, there you can manage wich will be displayed at wich position. Please note that you can put any number in that places, and they will be sorted from lowest number - top category to biggest number - last category.
+
+There are some **Additional Forum Options**. Inside them you can change some category settings. **Please note:**
+- **Forum link** will work and redirect to written url (in that case, note that you must write whole url as long as http:// or https://). And you will be able to still visit it's all forums and everything under it.
+- **Forum password will affect all subforums under that category.
+- **Forum is Active?** will **affect all subforums**
+- **Forum is Open?** will **not affect** subforums
+- **Style Options** will **affect all subforums** of that category **if** there is no different rule for specific subforum.
+- **Forum Rules** will **not affect** any subforum.
+- **Default** view options **will affect** all subforums of that category **if** there is no different rule for specific subforum.
+
+## Forums
+
+**Forums** and **subforums** are primary part of your software. Threads are placed inside them.
+
+Now, as stated above, they **must** be put under some Category. It's creation / editing page looks exactly the same like categories but with few differences.
+
+At forum you must **have Parent Forum** for every forum. That could be both category or forum.
+It is suggested to put some **Description** of each forum so users can easily navigate and know where to post thread.
+
+Everything else in that part is same as at categories.
+
+Next **different** thing than in categories is that all **Additional Forum Options** will affect that forum you are using / editing.
+
+Also, there is one cool thing that you can change forum permissions for each forum. But be careful with them and always check twice, it is your forum wich should be safe.
```

Comment:
I think this should be:

"Also, remember to change yourchange forum permissions for each forum. But be careful with them and always check twice, if these are set incorrectly you may give incorrect users access to private sections."